client/landing/stepper/declarative-flow/site-setup-flow.ts

```diff
@@ -446,6 +448,10 @@ export const siteSetupFlow: Flow = {
 				case 'intent':
 					return exitFlow( `/home/${ siteSlug }` );
 
+				case 'goals':
+					// Skip to dashboard must have been pressed
+					return exitFlow( `/home/${ siteSlug }`, true );
```

Comment:
Looks like we also need to set `isWpAdmin` to true at L449.

Reply:
I would prefer not do so unless you have strong arguments on why we should.  One of the aims here is not to change the current behaviour of the `intent` step. 

My **assumption** is that the `intent` screen works as expected at the moment, and does not need an intervention. It probably has the `intent` set to `wpadmin` in the onboard store, so there is no need for an extra flag on the exitFlow. Setting the `intent` to `wpadmin` in the onboard store was not desired in case of `goals` screen implementation.

Reply:
@arthur791004 FYI I have pinged @autumnfjeld on a related slack thread where I also cc'ed you: p1653343958317979-slack-C02TCEHP3HA

Reply:
If intent step's behaviour was "", then goals step should just follow "" (since goals behave like a drop in replacement for intent).

Reply:
Agree. And we can set the intent to `wpadmin` for both in the follow-up PR if we want.

Reply:
@arthur791004  @mashikag  I could see both sides of setting  `Skip to Dashboard`  to `intent=wpadmin` or  `intent=null`.

But I'm thinking null is the better way to go because it isn't the same as the other choices on the screen, it isn't as deliberate (or at least the design is meant to keep it less prominent compared to other options on the screen).

One "problem" we now have, or maybe "evolution" is a better term, is that intent is a loaded english word.  What we are really capturing now is the flow the user will proceed through. This is ok, we just have to keep that in mind.

When a user clicks "Skip to dashboard" we can track that through the skip button.  Though I'm kind of tempted to make `skip` an intent option. What do other people think?  No need to do this now, just an idea to discuss.  Maybe not a great idea. 🤣

Reply:
> When a user clicks "Skip to dashboard" we can track that through the skip button. Though I'm kind of tempted to make skip an intent option. What do other people think? No need to do this now, just an idea to discuss. Maybe not a great idea. 🤣

@autumnfjeld Interesting idea. Personally I think it is nice to have the `Skip to dashboard` button in the same place throughout all the steps of the onboard flow, i.e. the way it is currently designed. Having in-person usability tests could help to make such decisions.

Reply:
> Personally I think it is nice to have the Skip to dashboard button in the same place throughout all the steps 

Yes, for sure.  That should not change. :)